### ISSUE TITLE
Display spec file name on top of suite name

### ISSUE THREAD

**Issue by romankhomitskyi** [Shelex/cypress-allure-plugin#100]
**Describe the solution you'd like**
Display spec file name on top of suite name so that test results will be better structured when there are two spec files with the same describe 

This happens when you test one feature with different scenarios and split into different spec file to benefit from parallelization

**Comment by Shelex** [comment]
Hi @romankhomitskyi .
I was thinking about this feature for almost a year and was ready to celebrate a 1 year birthday of this request, almost ordered a cake for that purpose, however it seems we will have to close it as new [Suite structuring](https://github.com/Shelex/cypress-allure-plugin#suite-structuring) feature is available in v2.29.0 

The function with your custom logic of defining what should be a suite could be placed in `cypress/support/index.js` (for example in the same file where you do `import '@shelex/cypress-allure-plugin';`, in cypress v10+ it is `e2e.js`). 

So to make a filename a wrapper around your suites you can add this snippet:
```js
Cypress.Allure.reporter
    .getInterface()
    .defineSuiteLabels((titlePath, fileInfo) => {
        return [fileInfo.name, ...titlePath];
    });
```

In case you just wanted to leave things "as is" but just supplement suite name with file name:

 ```js
Cypress.Allure.reporter
    .getInterface()
    .defineSuiteLabels((titlePath, fileInfo) => {
        titlePath[0] = `${fileInfo.name} | ${titlePath[0]}`;
        return titlePath;
    });
```

Hope that would help!

**Comment by romankhomitskyi** [comment]
Cool, thanks
